### ISSUE TITLE
Exception when downloading d3dcompiler

### ISSUE THREAD

**Issue by MBR-0001** [truckersmp-cli/truckersmp-cli#238]
tldr: bug introduced in https://github.com/truckersmp-cli/truckersmp-cli/commit/4eab0c02187b9b372206e3442858e068554c1974#, check additional context

## Describe the bug
Using the `--activate-native-d3dcompiler` arg makes the launcher attempt to the d3d file but fails with `TypeError: expected str, bytes or os.PathLike object, not dict`

## Expected behavior
The launcher should download the file

## Command
~~~
truckersmp-cli -g ~/.steam/steam/steamapps/common/Euro\ Truck\ Simulator\ 2 -o ~/.steam/steam/compatibilitytools.d/Proton-6.16-GE-1 -x ~/.steam/steam/steamapps/compatdata/227300 --without-steam-runtime start ets2mp -d --activate-native-d3dcompiler
~~~

## Terminal output
```
** DEBUG **  Whether to setup native d3dcompiler_47: True
** DEBUG **  Downloading d3dcompiler_47.dll
** DEBUG **  Downloading file https://github.com/ImagingSIMS/ImagingSIMS/raw/162e4b02445c1fb621ce81c2bdf82a7870a3fd2a/Redist/x64/d3dcompiler_47.dll to /home/mbr/.local/share/truckersmp-cli/dlls
Traceback (most recent call last):
  File "/usr/bin/truckersmp-cli", line 19, in <module>
    main()
  File "/usr/lib/python3.9/site-packages/truckersmp_cli/main.py", line 182, in main
    starter.run()
  File "/usr/lib/python3.9/site-packages/truckersmp_cli/gamestarter.py", line 249, in run
    activate_native_d3dcompiler_47(prefix, args["wine"])
  File "/usr/lib/python3.9/site-packages/truckersmp_cli/utils.py", line 55, in activate_native_d3dcompiler_47
    if not download_files(
  File "/usr/lib/python3.9/site-packages/truckersmp_cli/utils.py", line 199, in download_files
    if not download_files(
  File "/usr/lib/python3.9/site-packages/truckersmp_cli/utils.py", line 175, in download_files
    dest["name"] = os.path.basename(dest["abspath"])
  File "/usr/lib/python3.9/posixpath.py", line 142, in basename
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not dict

```

## Screenshots
N/A

## Information
 - OS: Arch Linux
 - Proton or Wine version: Proton-6.16-GE-1
 - truckersmp-cli version: 0.7.2?

## Additional context
Looks like the code is providing a dict on line 201 (dest variable) but expects a path to store the file
https://github.com/truckersmp-cli/truckersmp-cli/blob/c96a44a7046498ca83359763681a20d459933bc7/truckersmp_cli/utils.py#L199-L202

Introduced in https://github.com/truckersmp-cli/truckersmp-cli/commit/4eab0c02187b9b372206e3442858e068554c1974#?

Bug can be bypassed by just downloading the file manually
